control-plane-operator/controllers/hostedcontrolplane/storage/assets/10_deployment-hypershift.yaml

```diff
@@ -0,0 +1,137 @@
+        - name: SNAPSHOTTER_IMAGE
+          value: quay.io/openshift/origin-csi-external-snapshotter:latest
+        - name: NODE_DRIVER_REGISTRAR_IMAGE
+          value: quay.io/openshift/origin-csi-node-driver-registrar:latest
+        - name: LIVENESS_PROBE_IMAGE
+          value: quay.io/openshift/origin-csi-livenessprobe:latest
+        - name: VSPHERE_PROBLEM_DETECTOR_OPERATOR_IMAGE
+          value: quay.io/openshift/origin-vsphere-problem-detector:latest
+        - name: AZURE_DISK_DRIVER_OPERATOR_IMAGE
+          value: quay.io/openshift/origin-azure-disk-csi-driver-operator:latest
+        - name: AZURE_DISK_DRIVER_IMAGE
+          value: quay.io/openshift/origin-azure-disk-csi-driver:latest
+        - name: AZURE_FILE_DRIVER_OPERATOR_IMAGE
+          value: quay.io/openshift/origin-azure-file-csi-driver-operator:latest
+        - name: AZURE_FILE_DRIVER_IMAGE
+          value: quay.io/openshift/origin-azure-file-csi-driver:latest
+        - name: KUBE_RBAC_PROXY_IMAGE
+          value: quay.io/openshift/origin-kube-rbac-proxy:latest
+        - name: VMWARE_VSPHERE_DRIVER_OPERATOR_IMAGE
+          value: quay.io/openshift/origin-vsphere-csi-driver-operator:latest
+        - name: VMWARE_VSPHERE_DRIVER_IMAGE
+          value: quay.io/openshift/origin-vsphere-csi-driver:latest
+        - name: VMWARE_VSPHERE_SYNCER_IMAGE
+          value: quay.io/openshift/origin-vsphere-csi-driver-syncer:latest
+        - name: CLUSTER_CLOUD_CONTROLLER_MANAGER_OPERATOR_IMAGE
+          value: quay.io/openshift/origin-cluster-cloud-controller-manager-operator:latest
+        - name: SHARED_RESOURCE_DRIVER_OPERATOR_IMAGE
+          value: quay.io/openshift/origin-csi-driver-shared-resource-operator:latest
+        - name: SHARED_RESOURCE_DRIVER_IMAGE
+          value: quay.io/openshift/origin-csi-driver-shared-resource:latest
+        - name: SHARED_RESOURCE_DRIVER_WEBHOOK_IMAGE
+          value: quay.io/openshift/origin-csi-driver-shared-resource-webhook:latest
+        - name: ALIBABA_DISK_DRIVER_OPERATOR_IMAGE
+          value: quay.io/openshift/origin-alibaba-disk-csi-driver-operator:latest
+        - name: ALIBABA_CLOUD_DRIVER_IMAGE
+          value: quay.io/openshift/origin-alibaba-cloud-csi-driver:latest
+        - name: IBM_VPC_BLOCK_DRIVER_OPERATOR_IMAGE
+          value: quay.io/openshift/origin-ibm-vpc-block-csi-driver-operator:latest
+        - name: IBM_VPC_BLOCK_DRIVER_IMAGE
+          value: quay.io/openshift/origin-ibm-vpc-block-csi-driver:latest
+        - name: IBM_VPC_NODE_LABEL_UPDATER_IMAGE
+          value: quay.io/openshift/origin-ibm-vpc-node-label-updater:latest
+        - name: POWERVS_BLOCK_CSI_DRIVER_OPERATOR_IMAGE
+          value: quay.io/openshift/origin-powervs-block-csi-driver-operator:latest
+        - name: POWERVS_BLOCK_CSI_DRIVER_IMAGE
+          value: quay.io/openshift/origin-powervs-block-csi-driver:latest
+        - name: HYPERSHIFT_IMAGE
+          value: tbd
```

Comment:
I thought we had this - quay.io/openshift/origin-control-plane:latest ?

Reply:
Fixed